website/src/pages/index.js

```diff
@@ -37,6 +38,8 @@ const HomepageHeader = () => {
           </Link>
         </div>
       </div>
+      <img className={clsx(styles.logo)} src="static/img/logo-light.png"></img>
+      <img className={clsx(styles.graphic1)} src="static/img/mojaGraphic1.png"></img>
```

Comment:
The assets fail to load in the Vercel preview. The correct `src` path is `img/<name>`.